src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalInternalUserFunctionalTest.java

```diff
@@ -1231,4 +1233,62 @@ private static List<Map<String, Object>> sortByValue(final List<Map<String, Obje
+    void updateContactInformationForOrganisationShouldReturnSuccess() {
+        log.info("updateContactInformationForOrganisationShouldReturnSuccess :: STARTED");
+
+        Map<String, Object> response = professionalApiClient.createOrganisation();
+        String organisationIdentifier = (String) response.get("organisationIdentifier");
+
+        assertThat(organisationIdentifier).isNotEmpty();
+        JsonPath orgResponse = professionalApiClient.retrieveOrganisationDetails(
+            organisationIdentifier, hmctsAdmin,OK);
+        assertNotNull(orgResponse);
+        List<HashMap>  existingContactInformationList =  (List) orgResponse.get("contactInformation");
+
+        assertThat(existingContactInformationList).isNotEmpty()
+            .hasSizeGreaterThan(0);
+
+        List<ContactInformationCreationRequest> contactInformationCreationRequestList =
+            professionalApiClient.createContactInformationCreationRequests();
+        assertThat(contactInformationCreationRequestList).isNotEmpty()
+            .hasSizeGreaterThan(0);
+        ContactInformationCreationRequest contactInformationCreationRequest =
+            contactInformationCreationRequestList.get(0);
+
+        existingContactInformationList.forEach(existingContactInfo -> {
+
+            assertThat(existingContactInfo.get("addressLine1")).isNotNull();
+            assertThat(existingContactInfo.get("addressLine2")).isNotNull();
+            assertThat(existingContactInfo.get("addressLine3")).isNotNull();
+            assertThat(existingContactInfo.get("uprn")).isNotNull();
+            assertThat(existingContactInfo.get("country")).isNotNull();
+            assertThat(existingContactInfo.get("townCity")).isNotNull();
+            assertThat(existingContactInfo.get("country")).isNotNull();
+            assertThat(existingContactInfo.get("postCode")).isNotNull();
+
+            Response result =
+                professionalApiClient.updateContactInformationsToOrganisation(aContactInformationCreationRequest()
+                    .uprn(contactInformationCreationRequest.getUprn())
+                    .addressLine1(contactInformationCreationRequest.getAddressLine1())
+                    .addressLine2(contactInformationCreationRequest.getAddressLine2())
+                    .addressLine3(contactInformationCreationRequest.getAddressLine3())
+                    .country(contactInformationCreationRequest.getCountry())
+                    .county(contactInformationCreationRequest.getCounty())
+                    .townCity(contactInformationCreationRequest.getTownCity())
+                    .postCode(contactInformationCreationRequest.getPostCode())
+                    .dxAddress(contactInformationCreationRequest.getDxAddress())
+                    .build(),
+                    OK,organisationIdentifier,true);
+
+            assertNotNull(result);
+            assertThat(result.getBody()).isNotNull();
+            assertThat(result.statusCode()).isEqualTo(200);
```

Comment:
add assertions to verify new address details are updated for each UPRN (if uprn is use ) or for each address id